build/entrypoint.execline

```diff
@@ -0,0 +1,79 @@
+#!/bin/execlineb -s1
+
+ifelse -n { eltest $1 = "cartesi-rollups-authority-claimer" } { exec $1 }
```

Comment:
Yes, it will catch the `cartesi-rollups-authority-claimer` provided as `CMD` do `docker run` and move forward executing the rest of the script.

I was unsure about the best place to put the file, do you have any suggestions?

Reply:
Got it! So, as we discussed offline, I'd rather not include the new entrypoint and `execline` altogether.

We should revert this and only use `cartesi-rollups-node` as CMD for `rollups-node-distroless`